connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -381,15 +381,15 @@ private void doTransitionTo(TargetState targetState) throws Throwable {
         }
     }
 
-    public boolean isSinkConnector() {
+    public final boolean isSinkConnector() {
```

Comment:
@C0urante Is this OK or are these classes public and subclassable? Similar for other changes affecting connect runtime classes.

Reply:
This is fine; none of the Connect- or MM2-related files touched in the PR are part of public API.

Reply:
Thanks @C0urante!